src/services/homework/hooks/index.js

```diff
@@ -97,124 +99,140 @@ const hasViewPermissionAfter = (hook) => {
 			return Promise.reject(new errors.Forbidden("You don't have permissions!"));
 		}
 	}
-	(hook.result.data) ? (hook.result.data = data) : (hook.result = data);
-	(hook.result.data) ? (hook.result.total = data.length) : (hook.total = data.length);
+	hook.result.data ? (hook.result.data = data) : (hook.result = data);
```

Comment:
Codacy found an issue: [Expected an assignment or function call and instead saw an expression.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)